search-commons/src/main/java/no/unit/nva/search2/model/ResourceParameterKey.java

```diff
@@ -53,15 +53,13 @@ public enum ResourceParameterKey implements ParameterKey {
     ORCID(STRING, "entityDescription.contributors.identity.orcId"),
     MODIFIED_BEFORE(DATE, LESS_THAN, "modifiedDate"),
     MODIFIED_SINCE(DATE, GREATER_THAN_OR_EQUAL_TO, "modifiedDate"),
-    PROJECT_CODE(STRING, "fundings.identifier"),
-    PUBLICATION_ID(STRING, "entityDescription.reference.publicationInstance.id"),
-    PUBLICATION_TYPE(STRING, "entityDescription.reference.publicationInstance.type"),
+    PROJECT(STRING, "projects.id"),
     PUBLISHED_BEFORE(DATE_STRING, LESS_THAN, "publishedDate"),
     PUBLISHED_SINCE(DATE_STRING, GREATER_THAN_OR_EQUAL_TO, "publishedDate"),
     TITLE(STRING, "entityDescription.mainTitle", 2F),
     UNIT(STRING, "entityDescription.contributors.affiliation.id"),
     USER(STRING, "resourceOwner.owner"),
-    YEAR_REPORTED(NUMBER, "entityDescription.publicationDate.year"),
+    PUBLICATION_YEAR(NUMBER, "entityDescription.publicationDate.year"),
```

Comment:
`PUBLICATION_YEAR(NUMBER, EQUALS, "entityDescription.publicationDate.year","(?i)year.?reported|publication.?year", null, null),`
we need to support 'year_reported' as well

Reply:
(?i)  = regex ignore case
 year.?reported -> year + any or no character + reported = yearReported /  year_reported / year reported / etc
 | -> OR
 publication.?year -> publication + any or no character + year = publicationYear/  publication_year/ publication year/ etc